tests/acceptance/conftest.py

```diff
@@ -73,3 +75,15 @@ def selenium(selenium, small_app):
     selenium.maximize_window()
     selenium.get(environ['SERVER_NAME'])
     return selenium
+
+
+@pytest.fixture()
+def login(selenium):
```

Comment:
Not sure if this is the best name, since this is doing the login (and logout) _as an admin_, and I imagine we will need at least another fixture for an unprivileged user. But YAGNI, so for now it can stay like this.